src/Monitor/Monitor/MonitorCmdletBase.cs

```diff
@@ -67,13 +67,19 @@ protected void WriteIdentifiedWarning(string cmdletName, string topic, string me
                 topic,
                 message);
 
-            if (withTimeStamp)
-            {
-                WriteWarningWithTimestamp(formattedMessage);
-            }
-            else
+            string supressWarningOrErrorValue = System.Environment.GetEnvironmentVariable(BreakingChangeAttributeHelper.SUPPRESS_ERROR_OR_WARNING_MESSAGE_ENV_VARIABLE_NAME);
```

Comment:
please check line 96 - 101, please remove the duplicate logic.

Reply:
Done, please check it.